inc/Settings/Settings.php

```diff
@@ -0,0 +1,164 @@
+<?php declare(strict_types = 1);
+
+namespace VIPRealTimeCollaboration\Settings;
+
+defined( 'ABSPATH' ) || exit();
+
+final class Settings {
+	private const SETTINGS_PAGE_SLUG = 'vip-real-time-collaboration-settings';
+	public const OPTION_NAME = 'vip_real_time_collaboration_settings';
+
+	public function __construct() {
+		add_action( 'admin_init', [ $this, 'register_settings' ] );
+		add_action( 'admin_menu', [ $this, 'add_options_page' ] );
+	}
+
+	public static function is_vip_rtc_enabled(): bool {
+		/** @var array<string> */
+		$options = get_option( self::OPTION_NAME, self::get_default_options() );
+
+		return isset( $options['enable-vip-rtc'] ) && (bool) $options['enable-vip-rtc'];
+	}
+
+	/**
+	 * Get the default options for the plugin.
+	 *
+	 * @return array The default options.
+	 */
+	public static function get_default_options(): array {
+		return [ 'enable-vip-rtc' => true ];
+	}
+
+	/**
+	 * Sanitize settings before saving.
+	 *
+	 * Converts radio button values ('1' for enabled, '0' for disabled) to boolean.
+	 *
+	 * @param array $input The input values from the form.
+	 * @return array The sanitized settings.
+	 * @psalm-suppress PossiblyUnusedMethod Psalm does not detect usage via add_filter.
+	 */
+	public static function sanitize_settings( ?array $input = [] ): array {
+		return [
+			'enable-vip-rtc' => isset( $input['enable-vip-rtc'] ) && '1' === $input['enable-vip-rtc'],
+		];
+	}
+
+	/**
+	 * Register the settings for the settings page
+	 */
+	public static function register_settings(): void {
+		register_setting(
+			self::SETTINGS_PAGE_SLUG,
+			self::OPTION_NAME,
+			[
+				'type' => 'array',
+				'default' => self::get_default_options(),
+				'sanitize_callback' => [ __CLASS__, 'sanitize_settings' ],
+			]
+		);
+
+		// Add instructions section.
+		add_settings_section(
+			'plugin-settings',
+			'',
+			[ __CLASS__, 'display_settings_instructions' ],
+			self::SETTINGS_PAGE_SLUG
+		);
+
+		/** @psalm-suppress InvalidArgument */ // WordPress Settings API allows custom args.
+		add_settings_field(
+			'enable-vip-rtc',
+			__( 'Real-Time Collaboration', 'vip-real-time-collaboration' ),
+			[ __CLASS__, 'display_settings_radio' ],
+			self::SETTINGS_PAGE_SLUG,
+			'plugin-settings',
+			[
+				'field_id' => 'enable-vip-rtc',
+				'enabled_label' => __( 'Enable real-time collaboration', 'vip-real-time-collaboration' ),
+				'disabled_label' => __( 'Disable real-time collaboration', 'vip-real-time-collaboration' ),
+				'description' => __( 'Disable real-time collaboration functionaliy in case of an emergency.', 'vip-real-time-collaboration' ),
```

Comment:
```suggestion
				'description' => __( 'Disable real-time collaboration functionality in case of an emergency.', 'vip-real-time-collaboration' ),
```

fixes spelling error